appinfo/info.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<info>
+	<id>GlobalSiteSelector</id>
+	<name>Global Site Selector</name>
+	<description>
+		The Global Site Selector allows you to run multiple small Nextcloud instances and redirect users to the right server
```

Comment:
Don't intend like this here as we render Markdown and this would be internally rendered as `<blockquote>`. So put this on one line please or use CDATA.